travis/cfg/travis_run_pylint_pr.cfg

```diff
@@ -0,0 +1,33 @@
+[MASTER]
```

Comment:
FYI this is a copy and paste of curreny cfg file but removing all enable checks.